src/NewComment.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import { postComment } from "./api";
+import { useParams } from "react-router-dom";
+import Comments from "./Comments";
+import "./NewComment.css";
+
+export default function NewComment() {
+	const { article_id } = useParams();
+	const [comment, setComment] = useState("");
+	const [postedComment, setPostedComment] = useState("");
+	function handleChange(e) {
+		setComment(e.target.value);
+	}
+	function handleSubmit(e) {
+		e.preventDefault();
+		postComment(article_id, comment).then((response) => {
+			if (response.message === "Network Error") {
```

Comment:
We shouldn't be handling errors inside our .then() - logically the error should never reach a .then() - have you toggled your network off and seen that this works? 0.0